### PR TITLE
Add usage vignette

### PR DESCRIPTION
**Pull request by rcannood**
## Minor changes

* Add vignette to showcase laminr usage.

### CODE REVIEW

File: vignettes/usage.Rmd [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+```{r connect}
+db <- connect("laminlabs/cellxgene")
+```
+
+
+### Print a LaminDB instance
+
+```{r print_instance}
+db
+```
```
Comment (lazappi) [inline vignettes/usage.Rmd:34]: Does printing need to be a separate section to getting the object?

File: vignettes/usage.Rmd [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+### Get module
+
```
Comment (lazappi) [inline vignettes/usage.Rmd:37]: Maybe each object type could have a one line description and a link to the Python docs?

**Review by lazappi — APPROVED** [review]
I think this is fine as a demonstration but I added some suggestions for more explanatory text. Up to you whether to add that now or later (maybe open an issue if you don't?).

**Comment by rcannood** [comment]
Thanks for the valuable feedback! I updated the usage with the additional information. Could you have a look? :)

**Review by lazappi — APPROVED** [review]

**Pull request merged**